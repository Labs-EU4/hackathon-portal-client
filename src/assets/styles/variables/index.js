export { Solid, Gradient } from './colors';
export {
    type,
    baseFontSize,
    smallFontSize,
    h1FontSize,
    h2FontSize,
    h3FontSize,
    h4FontSize,
    button_text
} from './fonts';
export { Spacing } from './spacing';
export { media } from './media';