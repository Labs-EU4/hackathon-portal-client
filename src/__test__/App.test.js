import React from "react";
import { Router } from "react-router";
import { createMemoryHistory } from "history";
import { Provider } from "react-redux";
import { render } from "@testing-library/react";
import configureStore from "redux-mock-store";
import "@testing-library/jest-dom/extend-expect";
import { initialState } from "../utils/mockData";
import App from "../App";
import { ThemeProvider } from "styled-components";
import { theme } from "../assets/styles/ThemeStyling";
import { library } from "@fortawesome/fontawesome-svg-core";
import { fas } from "@fortawesome/free-solid-svg-icons";
import { far } from "@fortawesome/free-regular-svg-icons";
import { fab } from "@fortawesome/free-brands-svg-icons";

library.add(fas, far, fab);

const history = createMemoryHistory();

let component;
let mockStore;
let store;

beforeEach(() => {
  mockStore = configureStore();
  store = mockStore(initialState);
  component = render(
    <Router history={history}>
      <Provider store={store}>
        <ThemeProvider theme={theme}>
          <App />
        </ThemeProvider>
      </Provider>
    </Router>
  );
});

describe("Shows all the text nodes on CreateTeam.js that are contained on the making an event form", () => {
  it("App.js component renders properly,with all the Private Routes", () => {
    expect(component).toMatchSnapshot();
  });

  it("The text node for the first character of the current user mail, renders properly on the menu", () => {
    expect(component.queryByText(/8/i)).toBeInTheDocument();
  });

  it("The text node <h4> for the EuroHack hackathon card, renders properly", () => {
    expect(component.queryByText(/Jake22/i)).toBeInTheDocument();
  });
  it("The text node <h4> for the World hackathon event card, renders properly", () => {
    expect(component.queryByText("I like coding")).toBeInTheDocument();
  });
  it("The text node <p> for the EuroHack hackathon event card description , renders properly", () => {
    expect(
      component.queryByText("Let's make your next hackathon a success!")
    ).toBeInTheDocument();
  });
  it("The text node <p> for the World hackathon event card description , renders properly", () => {
    expect(component.queryByText("8omemail@google.com")).toBeInTheDocument();
  });
  it("The text node <p> for the Eurohack hackathon event card date, renders properly", () => {
    expect(component.queryByText(/Create Event/i)).toBeInTheDocument();
  });
  it("The text node <p> for the World hackathon event card date, renders properly", () => {
    expect(
      component.getByText(/Meet our team of superstars/i)
    ).toBeInTheDocument();
  });
});
