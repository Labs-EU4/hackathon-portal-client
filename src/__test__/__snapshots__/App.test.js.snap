// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shows all the text nodes on CreateTeam.js that are contained on the making an event form App.js component renders properly,with all the Private Routes 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <main
        class="sc-EHOje gxePDH"
      >
        <div
          class="sc-htoDjs fkHpLM"
        >
          <a
            class="sc-jzJRlG eXJAOQ"
            href="/"
          >
            <img
              alt="Hackton - Organise hackathons"
              class="sc-dnqmqq jIqHVB"
              src="Hackton-logo.png"
            />
          </a>
          <div
            class="sc-fjdhpX gxARAQ"
          >
            <div
              class="sc-VigVT eJiYcu"
            >
              <button
                class="sc-iwsKbI fjbCBr sc-jTzLTM itrQvU"
                color="grey"
                data-testid="label"
              >
                Log Out
              </button>
            </div>
          </div>
        </div>
        <div
          class="sc-bZQynM ghjVHi"
        >
          <div
            class="sc-iBEsjs bQpzT"
          >
            <div
              class="sc-hzNEM cqQQaf"
            >
              <div
                class="sc-chbbiW dFjtDJ"
              >
                <img
                  alt="Hackton - Organise hackathons"
                  class="sc-dnqmqq jIqHVB"
                  src="Hackton-logo.png"
                />
                <div
                  class="sc-kxynE ejmOVk"
                >
                  Let's make your next hackathon a success!
                </div>
              </div>
            </div>
            <div
              class="sc-cooIXK bZulKE"
            >
              <h1
                class="sc-fcdeBU gLXibE"
              >
                Meet our team of superstars
                <hr />
              </h1>
              <div
                class="sc-gmeYpB dUEBOn"
              >
                <div
                  class="sc-kZmsYB emVFQE"
                />
                <div
                  class="sc-kZmsYB emVFQE"
                />
                <div
                  class="sc-kZmsYB emVFQE"
                />
                <div
                  class="sc-kZmsYB emVFQE"
                />
                <div
                  class="sc-kZmsYB emVFQE"
                />
                <div
                  class="sc-kZmsYB emVFQE"
                />
              </div>
            </div>
          </div>
          <div>
            <div
              class="sc-htoDjs fkHpLM"
            >
              <a
                class="sc-jzJRlG eXJAOQ"
                href="/"
              >
                <img
                  alt="Hackton - Organise hackathons"
                  class="sc-dnqmqq jIqHVB"
                  src="Hackton-logo.png"
                />
              </a>
              <div
                class="sc-fjdhpX gxARAQ"
              >
                <div
                  class="sc-VigVT eJiYcu"
                >
                  <button
                    class="sc-iwsKbI fjbCBr sc-jTzLTM itrQvU"
                    color="grey"
                    data-testid="label"
                  >
                    Log Out
                  </button>
                </div>
              </div>
            </div>
            <div
              class="sc-fAjcbJ eczLSz"
            >
              <div
                class="sc-bwzfXH ihQUfF sc-iGPElx dmxpbw"
              >
                <div
                  class="sc-jtRfpW fiPPEF"
                >
                  <h3
                    class="sc-eNQAEJ faMTgL"
                  >
                    Edit Profile
                  </h3>
                </div>
                <div
                  class="sc-dqBHgY fghFCM"
                >
                  <div
                    class="sc-cmTdod sc-jwKygS sc-hgHYgh fNjwQp"
                  >
                    <form
                      action="#"
                    >
                      <label
                        class="sc-bdVaJa sc-kasBVs cMZYBS"
                        for="image"
                      >
                        Profile picture
                      </label>
                      <figure
                        class="sc-jXQZqI hKQWpr"
                      >
                        <div>
                          <img
                            alt=""
                            src="profile-image.png"
                          />
                        </div>
                        <p>
                          Jake22
                        </p>
                      </figure>
                      <div
                        class="sc-dqBHgY fghFCM"
                      >
                        <label
                          class="sc-bdVaJa kxtZDY"
                          for="fullname"
                        >
                          Full Name
                        </label>
                        <input
                          class="sc-Rmtcm gqfbIr"
                          display="wide"
                          name="fullname"
                          placeholder="Full Name"
                          type="text"
                          value="Jake Tom"
                        />
                      </div>
                      <div
                        class="sc-dqBHgY fghFCM"
                      >
                        <label
                          class="sc-bdVaJa kxtZDY"
                          for="image"
                        >
                          Profile Image
                        </label>
                        <input
                          accept="image/*"
                          class="sc-Rmtcm gqfbIr"
                          display="wide"
                          name="image_url"
                          placeholder="Profile picture"
                          type="file"
                          value=""
                        />
                      </div>
                      <div
                        class="sc-dqBHgY fghFCM"
                      >
                        <div
                          class="sc-TFwJa ckHyee"
                        >
                          <label
                            class="sc-bdVaJa kxtZDY"
                            for="email"
                          >
                            Email
                          </label>
                          <input
                            class="sc-Rmtcm gqfbIr"
                            display="wide"
                            name="email"
                            placeholder="Email"
                            type="text"
                            value="8omemail@google.com"
                          />
                        </div>
                        <div
                          class="sc-TFwJa ckHyee"
                        >
                          <label
                            class="sc-bdVaJa kxtZDY"
                            for="username"
                          >
                            Username
                          </label>
                          <input
                            class="sc-Rmtcm gqfbIr"
                            display="wide"
                            name="username"
                            placeholder="Username"
                            type="text"
                            value="Jake22"
                          />
                        </div>
                      </div>
                      <div
                        class="sc-dqBHgY fghFCM"
                      >
                        <label
                          class="sc-bdVaJa kxtZDY"
                          for="bio"
                        >
                          Bio
                        </label>
                        <div
                          class="sc-clNaTc cGexw"
                        >
                          <textarea
                            name="bio"
                            placeholder="bio"
                          >
                            I like coding 
                          </textarea>
                        </div>
                      </div>
                      <div
                        class="sc-dqBHgY sc-eInJlc ecPLqu"
                      >
                        <a
                          href="/dashboard"
                        >
                          <button
                            class="sc-iwsKbI fjbCBr"
                            color="grey"
                            to="/dashboard"
                          >
                            Cancel
                          </button>
                        </a>
                        <button
                          class="sc-iwsKbI hdHLkz sc-gtfDJT hZNNQe"
                          color="green"
                          type="submit"
                        >
                          Save Changes
                        </button>
                      </div>
                    </form>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="sc-cSHVUG hNxZub"
            >
              <p
                class="sc-kGXeez gGPeLY"
              >
                International Crafters Â© 2020
              </p>
              <div>
                <a
                  class="sc-kAzzGY sc-kgoBCf dTiEqo"
                  href="#"
                >
                  Privacy
                </a>
                <a
                  class="sc-kAzzGY sc-kgoBCf dTiEqo"
                  href="#"
                >
                  Contacts
                </a>
              </div>
            </div>
          </div>
        </div>
        <div
          class="sc-cSHVUG hNxZub"
        >
          <p
            class="sc-kGXeez gGPeLY"
          >
            International Crafters Â© 2020
          </p>
          <div>
            <a
              class="sc-kAzzGY sc-kgoBCf dTiEqo"
              href="#"
            >
              Privacy
            </a>
            <a
              class="sc-kAzzGY sc-kgoBCf dTiEqo"
              href="#"
            >
              Contacts
            </a>
          </div>
        </div>
        <div
          class="sc-hmzhuo iikKYW"
        >
          <div
            class="sc-jbKcbu dgYNjB"
          >
            <div
              class="sc-frDJqD beTuwI"
            >
              <div
                alt="Jake22"
                class="sc-jqCOkK jOgQwK"
              />
              <div
                class="sc-ksYbfQ kSIYJi"
              >
                <p>
                  Jake Tom
                </p>
                <p>
                  8omemail@google.com
                </p>
              </div>
            </div>
          </div>
          <a
            activeclassname="current"
            class="sc-gZMcBi efUfxo sc-dNLxif lmjtRY"
            color="primary"
            href="/event/new"
          >
            Create
             Event
          </a>
          <div
            style="width: 100%; position: relative;"
          >
            <a
              class="sc-kvZOFW jpJKzP"
              href="/home"
            >
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-home fa-w-18 sc-gisBJw fevNjo"
                data-icon="home"
                data-prefix="fas"
                focusable="false"
                role="img"
                viewBox="0 0 576 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M280.37 148.26L96 300.11V464a16 16 0 0 0 16 16l112.06-.29a16 16 0 0 0 15.92-16V368a16 16 0 0 1 16-16h64a16 16 0 0 1 16 16v95.64a16 16 0 0 0 16 16.05L464 480a16 16 0 0 0 16-16V300L295.67 148.26a12.19 12.19 0 0 0-15.3 0zM571.6 251.47L488 182.56V44.05a12 12 0 0 0-12-12h-56a12 12 0 0 0-12 12v72.61L318.47 43a48 48 0 0 0-61 0L4.34 251.47a12 12 0 0 0-1.6 16.9l25.5 31A12 12 0 0 0 45.15 301l235.22-193.74a12.19 12.19 0 0 1 15.3 0L530.9 301a12 12 0 0 0 16.9-1.6l25.5-31a12 12 0 0 0-1.7-16.93z"
                  fill="currentColor"
                />
              </svg>
              <span>
                Home
              </span>
            </a>
          </div>
          <div
            style="width: 100%; position: relative;"
          >
            <a
              class="sc-kvZOFW jpJKzP"
              href="/dashboard"
            >
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-th-large fa-w-16 sc-gisBJw fevNjo"
                data-icon="th-large"
                data-prefix="fas"
                focusable="false"
                role="img"
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M296 32h192c13.255 0 24 10.745 24 24v160c0 13.255-10.745 24-24 24H296c-13.255 0-24-10.745-24-24V56c0-13.255 10.745-24 24-24zm-80 0H24C10.745 32 0 42.745 0 56v160c0 13.255 10.745 24 24 24h192c13.255 0 24-10.745 24-24V56c0-13.255-10.745-24-24-24zM0 296v160c0 13.255 10.745 24 24 24h192c13.255 0 24-10.745 24-24V296c0-13.255-10.745-24-24-24H24c-13.255 0-24 10.745-24 24zm296 184h192c13.255 0 24-10.745 24-24V296c0-13.255-10.745-24-24-24H296c-13.255 0-24 10.745-24 24v160c0 13.255 10.745 24 24 24z"
                  fill="currentColor"
                />
              </svg>
              <span>
                Dashboard
              </span>
            </a>
          </div>
          <div
            style="width: 100%; position: relative;"
          >
            <a
              aria-current="page"
              class="sc-kvZOFW jpJKzP current"
              href="/"
            >
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-info-circle fa-w-16 sc-gisBJw fevNjo"
                data-icon="info-circle"
                data-prefix="fas"
                focusable="false"
                role="img"
                viewBox="0 0 512 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M256 8C119.043 8 8 119.083 8 256c0 136.997 111.043 248 248 248s248-111.003 248-248C504 119.083 392.957 8 256 8zm0 110c23.196 0 42 18.804 42 42s-18.804 42-42 42-42-18.804-42-42 18.804-42 42-42zm56 254c0 6.627-5.373 12-12 12h-88c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h12v-64h-12c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h64c6.627 0 12 5.373 12 12v100h12c6.627 0 12 5.373 12 12v24z"
                  fill="currentColor"
                />
              </svg>
              <span>
                About
              </span>
            </a>
          </div>
          <div
            style="width: 100%; position: relative;"
          >
            <a
              class="sc-kvZOFW jpJKzP"
              href="/results"
            >
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-trophy fa-w-18 sc-gisBJw fevNjo"
                data-icon="trophy"
                data-prefix="fas"
                focusable="false"
                role="img"
                viewBox="0 0 576 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M552 64H448V24c0-13.3-10.7-24-24-24H152c-13.3 0-24 10.7-24 24v40H24C10.7 64 0 74.7 0 88v56c0 35.7 22.5 72.4 61.9 100.7 31.5 22.7 69.8 37.1 110 41.7C203.3 338.5 240 360 240 360v72h-48c-35.3 0-64 20.7-64 56v12c0 6.6 5.4 12 12 12h296c6.6 0 12-5.4 12-12v-12c0-35.3-28.7-56-64-56h-48v-72s36.7-21.5 68.1-73.6c40.3-4.6 78.6-19 110-41.7 39.3-28.3 61.9-65 61.9-100.7V88c0-13.3-10.7-24-24-24zM99.3 192.8C74.9 175.2 64 155.6 64 144v-16h64.2c1 32.6 5.8 61.2 12.8 86.2-15.1-5.2-29.2-12.4-41.7-21.4zM512 144c0 16.1-17.7 36.1-35.3 48.8-12.5 9-26.7 16.2-41.8 21.4 7-25 11.8-53.6 12.8-86.2H512v16z"
                  fill="currentColor"
                />
              </svg>
              <span>
                Results
              </span>
            </a>
          </div>
          <svg
            aria-hidden="true"
            class="svg-inline--fa fa-angle-double-down fa-w-10 sc-gisBJw fevNjo sc-cJSrbW dgKEVQ"
            data-icon="angle-double-down"
            data-prefix="fas"
            focusable="false"
            role="img"
            viewBox="0 0 320 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M143 256.3L7 120.3c-9.4-9.4-9.4-24.6 0-33.9l22.6-22.6c9.4-9.4 24.6-9.4 33.9 0l96.4 96.4 96.4-96.4c9.4-9.4 24.6-9.4 33.9 0L313 86.3c9.4 9.4 9.4 24.6 0 33.9l-136 136c-9.4 9.5-24.6 9.5-34 .1zm34 192l136-136c9.4-9.4 9.4-24.6 0-33.9l-22.6-22.6c-9.4-9.4-24.6-9.4-33.9 0L160 352.1l-96.4-96.4c-9.4-9.4-24.6-9.4-33.9 0L7 278.3c-9.4 9.4-9.4 24.6 0 33.9l136 136c9.4 9.5 24.6 9.5 34 .1z"
              fill="currentColor"
            />
          </svg>
        </div>
      </main>
      <div
        class="Toastify"
      />
    </div>
  </body>,
  "container": <div>
    <main
      class="sc-EHOje gxePDH"
    >
      <div
        class="sc-htoDjs fkHpLM"
      >
        <a
          class="sc-jzJRlG eXJAOQ"
          href="/"
        >
          <img
            alt="Hackton - Organise hackathons"
            class="sc-dnqmqq jIqHVB"
            src="Hackton-logo.png"
          />
        </a>
        <div
          class="sc-fjdhpX gxARAQ"
        >
          <div
            class="sc-VigVT eJiYcu"
          >
            <button
              class="sc-iwsKbI fjbCBr sc-jTzLTM itrQvU"
              color="grey"
              data-testid="label"
            >
              Log Out
            </button>
          </div>
        </div>
      </div>
      <div
        class="sc-bZQynM ghjVHi"
      >
        <div
          class="sc-iBEsjs bQpzT"
        >
          <div
            class="sc-hzNEM cqQQaf"
          >
            <div
              class="sc-chbbiW dFjtDJ"
            >
              <img
                alt="Hackton - Organise hackathons"
                class="sc-dnqmqq jIqHVB"
                src="Hackton-logo.png"
              />
              <div
                class="sc-kxynE ejmOVk"
              >
                Let's make your next hackathon a success!
              </div>
            </div>
          </div>
          <div
            class="sc-cooIXK bZulKE"
          >
            <h1
              class="sc-fcdeBU gLXibE"
            >
              Meet our team of superstars
              <hr />
            </h1>
            <div
              class="sc-gmeYpB dUEBOn"
            >
              <div
                class="sc-kZmsYB emVFQE"
              />
              <div
                class="sc-kZmsYB emVFQE"
              />
              <div
                class="sc-kZmsYB emVFQE"
              />
              <div
                class="sc-kZmsYB emVFQE"
              />
              <div
                class="sc-kZmsYB emVFQE"
              />
              <div
                class="sc-kZmsYB emVFQE"
              />
            </div>
          </div>
        </div>
        <div>
          <div
            class="sc-htoDjs fkHpLM"
          >
            <a
              class="sc-jzJRlG eXJAOQ"
              href="/"
            >
              <img
                alt="Hackton - Organise hackathons"
                class="sc-dnqmqq jIqHVB"
                src="Hackton-logo.png"
              />
            </a>
            <div
              class="sc-fjdhpX gxARAQ"
            >
              <div
                class="sc-VigVT eJiYcu"
              >
                <button
                  class="sc-iwsKbI fjbCBr sc-jTzLTM itrQvU"
                  color="grey"
                  data-testid="label"
                >
                  Log Out
                </button>
              </div>
            </div>
          </div>
          <div
            class="sc-fAjcbJ eczLSz"
          >
            <div
              class="sc-bwzfXH ihQUfF sc-iGPElx dmxpbw"
            >
              <div
                class="sc-jtRfpW fiPPEF"
              >
                <h3
                  class="sc-eNQAEJ faMTgL"
                >
                  Edit Profile
                </h3>
              </div>
              <div
                class="sc-dqBHgY fghFCM"
              >
                <div
                  class="sc-cmTdod sc-jwKygS sc-hgHYgh fNjwQp"
                >
                  <form
                    action="#"
                  >
                    <label
                      class="sc-bdVaJa sc-kasBVs cMZYBS"
                      for="image"
                    >
                      Profile picture
                    </label>
                    <figure
                      class="sc-jXQZqI hKQWpr"
                    >
                      <div>
                        <img
                          alt=""
                          src="profile-image.png"
                        />
                      </div>
                      <p>
                        Jake22
                      </p>
                    </figure>
                    <div
                      class="sc-dqBHgY fghFCM"
                    >
                      <label
                        class="sc-bdVaJa kxtZDY"
                        for="fullname"
                      >
                        Full Name
                      </label>
                      <input
                        class="sc-Rmtcm gqfbIr"
                        display="wide"
                        name="fullname"
                        placeholder="Full Name"
                        type="text"
                        value="Jake Tom"
                      />
                    </div>
                    <div
                      class="sc-dqBHgY fghFCM"
                    >
                      <label
                        class="sc-bdVaJa kxtZDY"
                        for="image"
                      >
                        Profile Image
                      </label>
                      <input
                        accept="image/*"
                        class="sc-Rmtcm gqfbIr"
                        display="wide"
                        name="image_url"
                        placeholder="Profile picture"
                        type="file"
                        value=""
                      />
                    </div>
                    <div
                      class="sc-dqBHgY fghFCM"
                    >
                      <div
                        class="sc-TFwJa ckHyee"
                      >
                        <label
                          class="sc-bdVaJa kxtZDY"
                          for="email"
                        >
                          Email
                        </label>
                        <input
                          class="sc-Rmtcm gqfbIr"
                          display="wide"
                          name="email"
                          placeholder="Email"
                          type="text"
                          value="8omemail@google.com"
                        />
                      </div>
                      <div
                        class="sc-TFwJa ckHyee"
                      >
                        <label
                          class="sc-bdVaJa kxtZDY"
                          for="username"
                        >
                          Username
                        </label>
                        <input
                          class="sc-Rmtcm gqfbIr"
                          display="wide"
                          name="username"
                          placeholder="Username"
                          type="text"
                          value="Jake22"
                        />
                      </div>
                    </div>
                    <div
                      class="sc-dqBHgY fghFCM"
                    >
                      <label
                        class="sc-bdVaJa kxtZDY"
                        for="bio"
                      >
                        Bio
                      </label>
                      <div
                        class="sc-clNaTc cGexw"
                      >
                        <textarea
                          name="bio"
                          placeholder="bio"
                        >
                          I like coding 
                        </textarea>
                      </div>
                    </div>
                    <div
                      class="sc-dqBHgY sc-eInJlc ecPLqu"
                    >
                      <a
                        href="/dashboard"
                      >
                        <button
                          class="sc-iwsKbI fjbCBr"
                          color="grey"
                          to="/dashboard"
                        >
                          Cancel
                        </button>
                      </a>
                      <button
                        class="sc-iwsKbI hdHLkz sc-gtfDJT hZNNQe"
                        color="green"
                        type="submit"
                      >
                        Save Changes
                      </button>
                    </div>
                  </form>
                </div>
              </div>
            </div>
          </div>
          <div
            class="sc-cSHVUG hNxZub"
          >
            <p
              class="sc-kGXeez gGPeLY"
            >
              International Crafters Â© 2020
            </p>
            <div>
              <a
                class="sc-kAzzGY sc-kgoBCf dTiEqo"
                href="#"
              >
                Privacy
              </a>
              <a
                class="sc-kAzzGY sc-kgoBCf dTiEqo"
                href="#"
              >
                Contacts
              </a>
            </div>
          </div>
        </div>
      </div>
      <div
        class="sc-cSHVUG hNxZub"
      >
        <p
          class="sc-kGXeez gGPeLY"
        >
          International Crafters Â© 2020
        </p>
        <div>
          <a
            class="sc-kAzzGY sc-kgoBCf dTiEqo"
            href="#"
          >
            Privacy
          </a>
          <a
            class="sc-kAzzGY sc-kgoBCf dTiEqo"
            href="#"
          >
            Contacts
          </a>
        </div>
      </div>
      <div
        class="sc-hmzhuo iikKYW"
      >
        <div
          class="sc-jbKcbu dgYNjB"
        >
          <div
            class="sc-frDJqD beTuwI"
          >
            <div
              alt="Jake22"
              class="sc-jqCOkK jOgQwK"
            />
            <div
              class="sc-ksYbfQ kSIYJi"
            >
              <p>
                Jake Tom
              </p>
              <p>
                8omemail@google.com
              </p>
            </div>
          </div>
        </div>
        <a
          activeclassname="current"
          class="sc-gZMcBi efUfxo sc-dNLxif lmjtRY"
          color="primary"
          href="/event/new"
        >
          Create
           Event
        </a>
        <div
          style="width: 100%; position: relative;"
        >
          <a
            class="sc-kvZOFW jpJKzP"
            href="/home"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-home fa-w-18 sc-gisBJw fevNjo"
              data-icon="home"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 576 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M280.37 148.26L96 300.11V464a16 16 0 0 0 16 16l112.06-.29a16 16 0 0 0 15.92-16V368a16 16 0 0 1 16-16h64a16 16 0 0 1 16 16v95.64a16 16 0 0 0 16 16.05L464 480a16 16 0 0 0 16-16V300L295.67 148.26a12.19 12.19 0 0 0-15.3 0zM571.6 251.47L488 182.56V44.05a12 12 0 0 0-12-12h-56a12 12 0 0 0-12 12v72.61L318.47 43a48 48 0 0 0-61 0L4.34 251.47a12 12 0 0 0-1.6 16.9l25.5 31A12 12 0 0 0 45.15 301l235.22-193.74a12.19 12.19 0 0 1 15.3 0L530.9 301a12 12 0 0 0 16.9-1.6l25.5-31a12 12 0 0 0-1.7-16.93z"
                fill="currentColor"
              />
            </svg>
            <span>
              Home
            </span>
          </a>
        </div>
        <div
          style="width: 100%; position: relative;"
        >
          <a
            class="sc-kvZOFW jpJKzP"
            href="/dashboard"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-th-large fa-w-16 sc-gisBJw fevNjo"
              data-icon="th-large"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M296 32h192c13.255 0 24 10.745 24 24v160c0 13.255-10.745 24-24 24H296c-13.255 0-24-10.745-24-24V56c0-13.255 10.745-24 24-24zm-80 0H24C10.745 32 0 42.745 0 56v160c0 13.255 10.745 24 24 24h192c13.255 0 24-10.745 24-24V56c0-13.255-10.745-24-24-24zM0 296v160c0 13.255 10.745 24 24 24h192c13.255 0 24-10.745 24-24V296c0-13.255-10.745-24-24-24H24c-13.255 0-24 10.745-24 24zm296 184h192c13.255 0 24-10.745 24-24V296c0-13.255-10.745-24-24-24H296c-13.255 0-24 10.745-24 24v160c0 13.255 10.745 24 24 24z"
                fill="currentColor"
              />
            </svg>
            <span>
              Dashboard
            </span>
          </a>
        </div>
        <div
          style="width: 100%; position: relative;"
        >
          <a
            aria-current="page"
            class="sc-kvZOFW jpJKzP current"
            href="/"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-info-circle fa-w-16 sc-gisBJw fevNjo"
              data-icon="info-circle"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M256 8C119.043 8 8 119.083 8 256c0 136.997 111.043 248 248 248s248-111.003 248-248C504 119.083 392.957 8 256 8zm0 110c23.196 0 42 18.804 42 42s-18.804 42-42 42-42-18.804-42-42 18.804-42 42-42zm56 254c0 6.627-5.373 12-12 12h-88c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h12v-64h-12c-6.627 0-12-5.373-12-12v-24c0-6.627 5.373-12 12-12h64c6.627 0 12 5.373 12 12v100h12c6.627 0 12 5.373 12 12v24z"
                fill="currentColor"
              />
            </svg>
            <span>
              About
            </span>
          </a>
        </div>
        <div
          style="width: 100%; position: relative;"
        >
          <a
            class="sc-kvZOFW jpJKzP"
            href="/results"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-trophy fa-w-18 sc-gisBJw fevNjo"
              data-icon="trophy"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 576 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M552 64H448V24c0-13.3-10.7-24-24-24H152c-13.3 0-24 10.7-24 24v40H24C10.7 64 0 74.7 0 88v56c0 35.7 22.5 72.4 61.9 100.7 31.5 22.7 69.8 37.1 110 41.7C203.3 338.5 240 360 240 360v72h-48c-35.3 0-64 20.7-64 56v12c0 6.6 5.4 12 12 12h296c6.6 0 12-5.4 12-12v-12c0-35.3-28.7-56-64-56h-48v-72s36.7-21.5 68.1-73.6c40.3-4.6 78.6-19 110-41.7 39.3-28.3 61.9-65 61.9-100.7V88c0-13.3-10.7-24-24-24zM99.3 192.8C74.9 175.2 64 155.6 64 144v-16h64.2c1 32.6 5.8 61.2 12.8 86.2-15.1-5.2-29.2-12.4-41.7-21.4zM512 144c0 16.1-17.7 36.1-35.3 48.8-12.5 9-26.7 16.2-41.8 21.4 7-25 11.8-53.6 12.8-86.2H512v16z"
                fill="currentColor"
              />
            </svg>
            <span>
              Results
            </span>
          </a>
        </div>
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-angle-double-down fa-w-10 sc-gisBJw fevNjo sc-cJSrbW dgKEVQ"
          data-icon="angle-double-down"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 320 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M143 256.3L7 120.3c-9.4-9.4-9.4-24.6 0-33.9l22.6-22.6c9.4-9.4 24.6-9.4 33.9 0l96.4 96.4 96.4-96.4c9.4-9.4 24.6-9.4 33.9 0L313 86.3c9.4 9.4 9.4 24.6 0 33.9l-136 136c-9.4 9.5-24.6 9.5-34 .1zm34 192l136-136c9.4-9.4 9.4-24.6 0-33.9l-22.6-22.6c-9.4-9.4-24.6-9.4-33.9 0L160 352.1l-96.4-96.4c-9.4-9.4-24.6-9.4-33.9 0L7 278.3c-9.4 9.4-9.4 24.6 0 33.9l136 136c9.4 9.5 24.6 9.5 34 .1z"
            fill="currentColor"
          />
        </svg>
      </div>
    </main>
    <div
      class="Toastify"
    />
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
